core/Mail.php

```diff
@@ -61,34 +63,76 @@ public function setWrappedHtmlBody($body)
     {
         $contentGenerator = StaticContainer::get(ContentGenerator::class);
         $bodyHtml = $contentGenerator->generateHtmlContent($body);
-        $this->setBodyHtml($bodyHtml);
+        $this->msgHTML($bodyHtml);
+    }
+
+    public function setBodyHtml($html)
+    {
+        $this->msgHTML($html);
+    }
+
+    /**
+     * Alias method for addAddress to keep BC to Zend_Mail
+     *
+     * @deprecated
+     * @param        $address
+     * @param string $name
+     * @throws \PHPMailer\PHPMailer\Exception
+     */
+    public function addTo($address, $name = '')
+    {
+        $this->addAddress($address, $name);
+    }
+
+    public function setBodyText($txt)
+    {
+        if ($this->ContentType == static::CONTENT_TYPE_TEXT_HTML) {
+            $this->AltBody = $txt;
+            return;
+        }
+
+        $this->Body = $txt;
     }
 
     /**
      * Sets the sender.
      *
      * @param string $email Email address of the sender.
      * @param null|string $name Name of the sender.
-     * @return Zend_Mail
+     * @return bool
      */
-    public function setFrom($email, $name = null)
+    public function setFrom($email, $name = null, $auto = true)
```

Comment:
could we document what the auto parameter does?

Reply:
will copy it over